roles/ovirt-host-deploy-libvirt-guests/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Fetch info about VDSM
+  command: bash -c "rpm -qi vdsm | grep -oE 'Version\\s+:\\s+[0-9\\.]+' | awk '{print $3}'"
+  changed_when: false
+  register: vdsm_version
+
+- block:
+    - name: Change default timeout if setup
+      lineinfile:
+        path: /etc/sysconfig/libvirt-guests
+        regexp: "^SHUTDOWN_TIMEOUT="
```

Comment:
If it's an INI file, why not use the INI files module instead of regexp?

Reply:
Yaniv, thanks for pointing this out. ini_file is indeed better suited for this, checked it and it works the same and it is more readable.